code/modules/mob/living/carbon/human/unarmed_attack.dm

```diff
@@ -90,7 +105,7 @@ var/global/list/sparring_attack_cache = list()
 				if(!target.lying)
 					target.visible_message("<span class='warning'>[target] gives way slightly.</span>")
 					target.apply_effect(attack_damage*3, PAIN, armour)
-	else if(attack_damage >= 5 && !(target == user) && (stun_chance + attack_damage * 5 >= 100) && armour < 1) // Chance to get the usual throwdown as well (25% standard chance)
+	else if(attack_damage >= 5 && !(target == user) && (stun_chance + attack_damage * 5 >= 100) && armour < 1 && (zone == BP_EYES || zone == BP_MOUTH || zone == BP_HEAD)) // Chance to get the usual throwdown as well (25% standard chance)
```

Comment:
```suggestion
	else if(attack_damage >= 5 && !(target == user) && (stun_chance + attack_damage * 5 >= 100) && armour < 1) // Chance to get the usual throwdown as well (25% standard chance)
```

Reply:
Should get rid of this, or only head punches will knock people down regardless of CQC skill